cockroachcloud/quickstart.md

```diff
@@ -219,80 +226,42 @@ Use the [JDBC driver](https://jdbc.postgresql.org/) in a Java application.
 <section class="filter-content" markdown="1" data-scope="java">
 
 <section class="filter-content" markdown="1" data-scope="mac">
-1. Use the `cockroach convert-url` command to convert the connection string that you copied earlier to a [valid connection string for JDBC connections](../{{site.versions["stable"]}}/connect-to-the-database.html?filters=java):
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    cockroach convert-url --url "<connection-string>"
-    ~~~
-
-    ~~~
-    ...
-
-    # Connection URL for JDBC (Java and JVM-based languages):
-    jdbc:postgresql://{host}:{port}/{database}?options=--cluster%3D{routing-id}&password={password}&sslmode=verify-full&user={username}
-    ~~~
+In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:
 
-1. Set the `JDBC_DATABASE_URL` environment variable to the JDBC-compatible connection string:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    export JDBC_DATABASE_URL="<jdbc-connection-string>"
-    ~~~
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export JDBC_DATABASE_URL="<jdbc-connection-string>"
+~~~
 
-    The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
+The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
 </section>
 
 <section class="filter-content" markdown="1" data-scope="linux">
-1. Use the `cockroach convert-url` command to convert the connection string that you copied earlier to a [valid connection string for JDBC connections](../{{site.versions["stable"]}}/connect-to-the-database.html?filters=java):
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    cockroach convert-url --url "<connection-string>"
-    ~~~
+In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:
```

Comment:
```suggestion
In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:
```
```suggestion
In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied earlier:
```

Reply:
GitHub has been doing a weird thing with duplicating suggestions... Ignore the empty suggestion above